test/main_test.js

```diff
@@ -152,9 +152,10 @@ suite('End to End', () => {
     let shoulds = [
       'schemas/foo.json',
       'schemas/bar.json',
+      'docs/documenting-non-services.md',
```

Comment:
what was behind this change?

Reply:
It turns out that the test was "passing" only due to an `unhandled rejection` not being fatal. When I was running the tests I noticed a bunch of logging about that and added it.